kubernetes/cpi.yml

```diff
@@ -0,0 +1,95 @@
+---
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-kubernetes-cpi
+    url: https://github.com/SAP/bosh-kubernetes-cpi-release/releases/download/v3-alpha/bosh-kubernetes-cpi-3-alpha.tgz
+    sha1: 5124702e611bb22181ed348d80324d3244424105
+
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: https://github.com/SAP/bosh-kubernetes-cpi-release/releases/download/v1-alpha/bosh-stemcell-3363.9-kubernetes-ubuntu-trusty-go_agent.tgz
+    sha1: 319acc5aa020f12a2a20392c971ebf8ef3d0cd6e
+
+# Configure sizes
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    services:
+      - name: outer-bosh
+        type: NodePort
+        ports:
+          - name: ssh-gateway
+            port: 22
+            nodePort: 30022
+          - name: bosh-init
+            port: 6868
+            nodePort: 30068
+          - name: nats
+            port: 4222
+            nodePort: 30042
+          - name: blobstore
+            port: 30250
+            nodePort: 30250
+          - name: director
+            port: 25555
+            nodePort: 30555
+
+- type: replace
+  path: /resource_pools/name=vms/env?/bosh?/authorized_keys?/-
+  value:
+    ((vcap_public_key))
+
+- type: remove
+  path: /instance_groups/name=bosh/networks/name=default/static_ips
+
+- type: replace
+  path: /networks/name=default
+  value:
+    name: default
+    type: dynamic
+
+# Add CPI job
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value: &cpi_job
+    name: kubernetes_cpi
+    release: bosh-kubernetes-cpi
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/director/cpi_job?
+  value: kubernetes_cpi
+
+- type: replace
+  path: /cloud_provider/template?
+  value: *cpi_job
+
+- type: replace
+  path: /disk_pools/name=disks/disk_size
+  value: 10_000
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/blobstore/port
+  value: 30250
+- type: replace
+  path: /instance_groups/name=bosh/properties/agent/mbus
+  value: nats://nats:((nats_password))@((internal_ip)):30042
+- type: replace
+  path: /cloud_provider/mbus
+  value: https://mbus:((mbus_bootstrap_password))@((internal_ip)):30068
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/kubernetes?
+  value:
+    cluster-access: ServiceAccount
+
+- type: replace
+  path: /cloud_provider/properties/kubernetes?
+  value:
+    cluster-access:
+      server: ((kube_api_server))
+      namespace: ((kube_namespace))
+      credentials:
+        certificate: ((client_cert))
+        private-key: ((client_key))
```

Comment:
im not too hot on casing differences. there are dashes, underscores and camel case that's involved. let's convert everything to underscores.